decidim-conferences/app/queries/decidim/conferences/organization_conferences.rb

```diff
@@ -9,7 +9,7 @@ def initialize(organization)
       end
 
       def query
-        Decidim::Conference.where(organization: @organization)
+        Decidim::Conference.where(organization: @organization).order(weight: :asc)
```

Comment:
Could we add a secondary order by date in case they have the same weights (as in after an upgrade)?
```suggestion
        Decidim::Conference.where(organization: @organization).order(weight: :asc, start_date: :desc)
```